BlackNectar/Carthage/Checkouts/AromaSwiftClient/.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: objective-c
+cache: cocoapods
+rvm: 2.2.2
+osx_image: xcode8
+before_install:
+  - brew update
+
+  # Only if app has Carthage dependencies
+  - carthage bootstrap --verbose
+
+script:
+  - carthage build --no-skip-current
+  - |
+    xcodebuild test \
```

Comment:
It looks like the project and scheme are not pointing to the right project. This will cause the build to fail.